connectivity/drivers/lora/TARGET_STM32WL/STM32WL_LoRaRadio.cpp

```diff
@@ -0,0 +1,1666 @@
+
+/**
+ / _____)             _              | |
+( (____  _____ ____ _| |_ _____  ____| |__
+ \____ \| ___ |    (_   _) ___ |/ ___)  _ \
+ _____) ) ____| | | || |_| ____( (___| | | |
+(______/|_____)_|_|_| \__)_____)\____)_| |_|
+    (C)2015 Semtech
+ ___ _____ _   ___ _  _____ ___  ___  ___ ___
+/ __|_   _/_\ / __| |/ / __/ _ \| _ \/ __| __|
+\__ \ | |/ _ \ (__| ' <| _| (_) |   / (__| _|
+|___/ |_/_/ \_\___|_|\_\_| \___/|_|_\\___|___|
+embedded.connectivity.solutions===============
+
+Description: LoRaWAN stack layer that controls both MAC and PHY underneath
+
+License: Revised BSD License, see LICENSE.TXT file include in the project
+
+Maintainer: Miguel Luis, Gregory Cristian & Gilbert Menth
+
+Copyright (c) 2019, Arm Limited and affiliates.
+
+SPDX-License-Identifier: BSD-3-Clause
+  */
+/**
+  ******************************************************************************
+  *
+  *          Portions COPYRIGHT 2021 STMicroelectronics
+  *
+  * @file    STM32WL_LoRaRadio.c
+  * @author  MCD Application Team
+  * @brief   radio driver implementation
+  ******************************************************************************
+  */
+
+
+#include <math.h>
+#include "ThisThread.h"
+#include "mbed_wait_api.h"
+#include "Timer.h"
+#include "STM32WL_LoRaRadio.h"
+#include "stm32wlxx_hal_subghz.h"
```

Comment:
seems that only 
#include <math.h>
#include "Timer.h"
#include "STM32WL_LoRaRadio.h"
is useful ?